.gitignore

```diff
@@ -106,3 +106,4 @@ Generated_Code #added for RIA/Silverlight projects
 _UpgradeReport_Files/
 Backup*/
 UpgradeLog*.XML
+/.vs
```

Comment:
the .vs folder is a cache for Visual Studio 2015 and Visual Studio 2017. It should be ignored.